src/crud.py

```
@@ -436,6 +445,157 @@ async def import_csv_data(db: AsyncSession, csv_file_path: str, overwrite_existi
             "message": "Import failed"
         }
 
+async def import_csv_data_into_dataset(db: AsyncSession, csv_file_path: str, dataset_id: int, overwrite_existing: bool = False) -> Dict[str, Any]:
```

Pylint (3.3.7)

[refactor] Too many local variables (19/15)

(R0914)